core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -453,7 +453,10 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
         }
     )(use)
 
-  final def managed(release: A => UIO[_]): ZManaged[R, E, A] =
+  /**
+   * Converts this ZIO to [[scalaz.zio.Managed]].
+   */
+  final def toManaged(release: A => UIO[_]): ZManaged[R, E, A] =
```

Comment:
Why do you rename this method? I think `managed` fits other combinators better than `toManaged`, although this one is unique because it returns totally different type.